lib/tarpon.rb

```diff
@@ -9,6 +9,9 @@
 require 'tarpon/request/subscriber'
 require 'tarpon/request/subscriber/entitlement'
 require 'tarpon/request/subscriber/subscription'
+require 'tarpon/request/subscriber/offering'
 require 'tarpon/request/receipt'
 require 'tarpon/response'
 require 'tarpon/version'
+
+API_VERSION = 'v3.0'
```

Comment:
what is this being used for?

Reply:
It is for docuemntation. They have different API versions. If you are so against I will delete

Reply:
Perhaps that can be done in another PR with this specific purpose, actually using this value interpolating in the actual endpoints.